testing/tests/named_filter_arguments.rs

```diff
@@ -0,0 +1,188 @@
+use askama::Template;
+
+#[test]
+fn test_pluralize_two_positional_args() {
```

Comment:
So many missing code comments describing what the test is checking. :stuck_out_tongue:

Reply:
The function name describes it, doesn't it?

Reply:
Yeah true, nitpicking a bit too much here maybe. ^^'